lnwallet/channel.go

```diff
@@ -4777,7 +4787,8 @@ func (lc *LightningChannel) availableBalance() (lnwire.MilliSatoshi, int64) {
 
 	// Next we'll grab the current set of log updates that are still active
 	// and haven't been garbage collected.
-	htlcView := lc.fetchHTLCView(lc.remoteUpdateLog.logIndex,
+	remoteACKedIndex := lc.localCommitChain.tail().theirMessageIndex
```

Comment:
Also, in `SignNextCommitment`, `tail()` is used here. Should it be `tip()`? 🤔

Reply:
In the spirit of this change, as it's attempting to increase the fidelity of our estimates, I'd say it should be: `tip`. The `SignNextCommitment`, `tail()` is correct as we currently only allow a single unrevoked commitment. Once we move back to allowing multiple unrevoked, then it can simply be `tip`.